src/main/resources/db/migration/V135__amend_facilities_dropdown_options.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE search_facility SET name = 'Loop Hearing' WHERE name LIKE 'Hearing%';
+UPDATE search_facilityfacilitytype SET facility_type_id = 36 WHERE facility_type_id = 56;
```

Comment:
One thing I always wonder about is whether Id's can get changed within future migrations at all. For example, if another migration in the future switches up some records...adds...removes etc 

If thats a concern, one idea would be to do something like 

```
UPDATE search_facilityfacilitytype SET facility_type_id = (select facility_type_id from search_facilityfacilitytype sfft join search_facility sf on sf.facility_id = sfft.facility_id where name = 'Hearing Loop' ) WHERE facility_type_id =  (select facility_type_id from search_facilityfacilitytype sfft join search_facility sf on sf.facility_id = sfft.facility_id where name = 'Hearing Loop T' );
```